src/util/psbt/map/input.rs

```diff
@@ -155,6 +157,34 @@ pub struct PsbtSigHashType {
     pub (in ::util::psbt) inner: u32,
 }
 
+impl fmt::Display for PsbtSigHashType {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self.ecdsa_hash_ty() {
```

Comment:
What happens in the case when we have `SchnorrSigHashType::Default`

Reply:
This is not related to this PR and a question to how `ecdsa_hash_ty` function is working. Will check later, but I assume the correct behaviour should be to return `ALL` value.

Reply:
I think this is related to this PR. The display for PsbtSigHashType should cover the `00` value case which is `Default`. Note that this variant is not present in EcdsaSigHashType

Reply:
I believe it gets output as `0x00` - I wondered the same thing :)